GVRf/Framework/src/org/gearvrf/GVRMaterial.java

```diff
@@ -214,6 +219,36 @@ public void setMainTexture(Future<GVRTexture> texture) {
         setTexture(MAIN_TEXTURE, texture);
     }
 
+    // ---------------------------------------------AR------------------------{
+    public void setAR(int ar) {
+        setAR(MAIN_TEXTURE, ar);
+    }
+
+    public void setMVPmat(float MVPmatrix[]) {
+        setMVPmatrix(MAIN_TEXTURE, MVPmatrix);
+    }
+
+    public void setAROffsets() {
+        setAROffsets(MAIN_TEXTURE, ar_offsets);
+    }
+
+    public void setAR(String key, int ar) {
+        checkStringNotNullOrEmpty("key", key);
+        NativeMaterial.setAR(getNative(), key, ar);
+    }
+
+    public void setMVPmatrix(String key, float MVPmatrix[]) {
+        checkStringNotNullOrEmpty("key", key);
```

Comment:
The naming for setMVPmat and setMVPmatrix, also with setAR and setATOffsets are really confusing for developers to understand.